packages/backend/src/payment-method/ilp/connector/core/controllers/stream.ts

```diff
@@ -23,9 +23,9 @@ export function createStreamController(): ILPMiddleware {
       return
     }
 
-    const moneyOrReply = streamServer.createReply(request.prepare)
+    const moneyOrReply = streamServer.createReply(request.prepare) // Creates a reply
```

Comment:
im not sure what context the comment is adding here since its basically just repeating the method name.... add more context or remove?